### DISCUSSION TITLE
6 digit code for final(i think) verification never comes

### DISCUSSION THREAD

**Discussion by Emanresu513z** [community/community#144205] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Github 6 digy code -says it was sent to"my device"I have asked for it twice and checked all "devices" , 2 phones, all email accounts. Really frustrating I can't get work I haves setting. Any ideas people???

**Comment by gr1shan1a** [comment]
try GitHub mobile or use better 2FA with verify

for easy use - [link to goole extention(2fa-authentification)](https://chromewebstore.google.com/detail/2fa-authenticator-app/gmohoglkppnemohbcgjakmgengkeaphi)
